### PR TITLE
Feature add hotspot content type

### PR DESCRIPTION
**Pull request by hlibmykhailov**

### CODE REVIEW

File: src/components/contents/ContentItem/HotspotOnImage/HotspotOnImage.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    this.setState({
+      [id]: !this.state[id] || false
+    });
+    classList.toggle('active');
+  };
+
+  popoverCloseHandler = event => {
+    let { id, classList } = event.target;
+    this.setState({
+      [id]: false
```
Comment (oleksandrbulakh) [inline src/components/contents/ContentItem/HotspotOnImage/HotspotOnImage.js]: remove brackets

File: src/components/contents/ContentItem/HotspotOnImage/HotspotOnImage.styled.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  position: absolute;
+  display: block;
+  width: ${props => (parseFloat(props.width) * 100 / parseFloat(props.naturalWidth)).toFixed(4)}%;
+  height: ${props =>
+    (parseFloat(props.height) * 100 / parseFloat(props.naturalHeight)).toFixed(4)}%;
+  top: ${props => (parseFloat(props.top) * 100 / parseFloat(props.naturalHeight)).toFixed(4)}%;
+  left: ${props => (parseFloat(props.left) * 100 / parseFloat(props.naturalWidth)).toFixed(4)}%;
+`;
+
+export const PopoverInfo = styled.button`
```
[the remark this answers is not in the document]
Comment (hlibmykhailov) [reply src/components/contents/ContentItem/HotspotOnImage/HotspotOnImage.styled.js]: Done

File: src/components/contents/ContentItem/HotspotOnImage/HotspotOnImage.styled.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  animation: pulse 1.5s infinite;
+  transform: translate(-50%, -50%);
+  padding: 0;
+  outline: none;
+  border: none;
```
[the remark this answers is not in the document]
Comment (hlibmykhailov) [reply src/components/contents/ContentItem/HotspotOnImage/HotspotOnImage.styled.js]: Done

File: src/components/contents/ContentItem/HotspotOnImage/HotspotOnImage.styled.js [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+import { transparentize } from 'utils/color';
+import * as typography from 'components/mixins/typography';
+import media from 'components/mixins/media';
+
+export const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  transition: all 0.3s ease-in-out;
```
[the remark this answers is not in the document]
Comment (hlibmykhailov) [reply src/components/contents/ContentItem/HotspotOnImage/HotspotOnImage.styled.js]: Done

File: src/components/contents/ContentItem/HotspotOnImage/HotspotOnImage.styled.js [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    border-radius: 50%;
+  }
+
+  &:hover,
+  &:focus {
+    cursor: pointer;
```
[the remark this answers is not in the document]
Comment (hlibmykhailov) [reply src/components/contents/ContentItem/HotspotOnImage/HotspotOnImage.styled.js]: Done

File: src/components/contents/ContentItem/HotspotOnImage/HotspotOnImage.styled.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (hlibmykhailov) [reply src/components/contents/ContentItem/HotspotOnImage/HotspotOnImage.styled.js]: Done

**Pull request merged**